_config.yml

```diff
@@ -10,10 +10,14 @@
 #  dc: Data Carpentry
 #  lc: Library Carpentry
 #  cp: Carpentries (to use for instructor training for instance)
+#  incubator: for Carpentries Incubator lesson pilot workshops
```

Comment:
thinking a little bit more about this, I'm wondering whether we should focus this more towards the notion of "pilot workshops" in general rather than incubator pilots? the pilot could be incubator lessons but we could also use the template when we are running new official curricula (e.g. astronomy).

Reply:
So `"pilot"` rather than `"incubator"` here and elsewhere? Would we still use the egg logo? Or something else?